packages/server/src/api/controllers/static/index.ts

```diff
@@ -167,10 +222,23 @@ export const serveApp = async function (ctx: Ctx) {
     const appInfo = await db.get<any>(DocumentType.APP_METADATA)
     let appId = context.getAppId()
 
+    const hideDevTools = !!ctx.params.appUrl
+    const sideNav = appInfo.navigation.navigation === "Left"
+    const hideFooter = ctx.user?.license?.features?.includes("branding")
```

Comment:
We should try to use `isFeatureEnabled` from `packages/server/src/features.ts`

Reply:
I'm a little confused by this,  this was throwing a ts build error that I just fixed, and I'm pretty sure the types it's expecting are defined [here])https://github.com/Budibase/budibase/blob/app-skeleton-2/packages/types/src/sdk/licensing/feature.ts) which don't seem to be referenced in the file the function you're referring to is defined in. The features defined there don't seem to match onto the ones defined in the linked types file either.